app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/ProductDetailsPage.scala.html

```diff
@@ -56,6 +51,12 @@
 
 @mainTemplate(title = messages("search-software.title"), breadcrumbs = Some(pageBreadcrumbs)) {
 
-  <p> Product details for software vendor "@{softwareVendor}" </p>
+  <dl class="govuk-summary-list contact-details">
+    <div class="govuk-summary-list__row">
+      <dd class="govuk-summary-list__value govuk-!-padding-bottom-4">
+        Email: <a class="govuk-link" href=@{s"mailto:${softwareVendor.email}"}>@{softwareVendor.email}</a>
```

Comment:
Email:
should be in a message ideally otherwise we might miss this when we need to add welsh messages